src/language-vue/parser-vue.js

```diff
@@ -18,7 +18,8 @@ const { hasPragma } = require("./pragma");
 function makeMap(str, expectsLowerCase) {
   const map = Object.create(null);
   const list = str.split(",");
-  for (let i = 0; i < list.length; i++) {
+  const listLength = list.length;
+  for (let i = 0; i < listLength; i++) {
```

Comment:
[The optimization seems not obvious and it's not easy to compare ](https://github.com/prettier/prettier/pull/4842#issuecomment-405432668) so I think we should preserve the original one here for readability.